### PR TITLE
docs: add `brew install hashicorp/tap/consul-k8s` and re-order install and uninstall workflows 

### PR DESCRIPTION
**Pull request by david-yu**
Add brew install workflow to docs, and re-order Consul K8s CLI to come first.

### CODE REVIEW

File: website/content/docs/k8s/installation/install.mdx [hunk 1]
```diff
@@ -17,20 +17,69 @@ a server running inside or outside of Kubernetes.
...
 For a hands-on experience with Consul as a service mesh
 for Kubernetes, follow the [Getting Started with Consul service
 mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
+## Consul K8s CLI Installation
+
+The recommended way to install Consul on Kubernetes for single cluster use cases is via the [Consul K8S CLI](/docs/k8s/k8s-cli). 
+You can install Consul on Kubernetes using the Consul K8s CLI tool after installing the CLI. 
+Ensure that `kubectl` is already configured to authenticate to the Kubernetes cluster using a valid `kubeconfig` file. 
+The Consul K8s CLI is currently available as an beta release and is not recommended for production environments.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/install.mdx:33]:
```suggestion
We recommend using the [Consul K8S CLI](/docs/k8s/k8s-cli) to install Consul on Kubernetes for single-cluster deployments. You can install Consul on Kubernetes using the Consul K8s CLI tool after installing the CLI. 

-> **Consul K8s CLI is currently in beta**: The tools is not recommended for production environments.

Before beginning the installation process, verify that `kubectl` is already configured to authenticate to the Kubernetes cluster using a valid `kubeconfig` file. 

The [Homebrew](https://brew.sh) package manager is required to complete the following installation instructions.
```
Just trying to make the language a little cleaner; also mentioned that the CLI tool installation procedure uses brew.

File: website/content/docs/k8s/installation/install.mdx [hunk 2]
```diff
@@ -17,20 +17,69 @@ a server running inside or outside of Kubernetes.
...
+
+1. First, install the HashiCorp tap, a repository of all our Homebrew packages.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/install.mdx:35]:
```suggestion
1. Install the HashiCorp `tap`, which is a repository of all Homebrew packages for HashiCorp:
```

File: website/content/docs/k8s/installation/install.mdx [hunk 3]
```diff
@@ -17,20 +17,69 @@ a server running inside or outside of Kubernetes.
...
+
 ## Helm Chart Installation
 
-The recommended way to run Consul on Kubernetes is via the
-[Helm chart](/docs/k8s/helm). This will install and configure
-all the necessary components to run Consul. The configuration enables you
-to run a server cluster, a client cluster, or both.
+For multi-cluster installs that involve cross-partition of cross datacenter communication, installing via the Consul on Kubernetes 
+Helm chart is the recommended process. This will install and configure all the necessary components to run Consul. The configuration 
+enables you to run a server cluster, a client cluster, or both.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/install.mdx:82]:
```suggestion
We recommend using the Consul Helm chart to install Consul on Kubernetes for multi-cluster installations that involve cross-partition of cross datacenter communication. The Helm chart installs and configures all necessary components to run Consul. The configuration enables you to run a server cluster, a client cluster, or both.
```
Making the language consistent with the previous section.

File: website/content/docs/k8s/k8s-cli.mdx [hunk 4]
```diff
@@ -12,26 +12,29 @@ This topic describes the commands, subcommands, and available options for using
 
 ~> Consul K8s CLI is is currently availabe as a beta release and is not recommended for production environments.
 
-## Download and Build the Binary
+## Install the Consul K8s CLI
 
-1. Clone the [consul-k8s](https://github.com/hashicorp/consul-k8s/tree/main/cli) project.
-1. Navigate to the `consul-k8s/cli` directory and issue the following command to build the tool:
+1. First, install the HashiCorp tap, a repository of all our Homebrew packages.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/k8s-cli.mdx:17]:
```suggestion
The [Homebrew](https://brew.sh) package manager is required to complete the following installation instructions.

1. Install the HashiCorp `tap`, which is a repository of all Homebrew packages for HashiCorp:
```
For consistency

File: website/content/docs/k8s/k8s-cli.mdx [hunk 5]
```diff
@@ -12,26 +12,29 @@ This topic describes the commands, subcommands, and available options for using
...
+    ```shell-session
+    brew tap hashicorp/tap
+    ```
+    
+1. Next, install the Consul K8s CLI with `hashicorp/tap/consul` formulae.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/k8s-cli.mdx:22]:
```suggestion
1. Install the Consul K8s CLI with `hashicorp/tap/consul` formula. 
```
For consistency

File: website/content/docs/k8s/installation/install.mdx [hunk 6]
```diff
@@ -17,20 +17,69 @@ a server running inside or outside of Kubernetes.
...
+You can install Consul on Kubernetes using the Consul K8s CLI tool after installing the CLI. 
+Ensure that `kubectl` is already configured to authenticate to the Kubernetes cluster using a valid `kubeconfig` file. 
+The Consul K8s CLI is currently available as an beta release and is not recommended for production environments.
+
+1. First, install the HashiCorp tap, a repository of all our Homebrew packages.
+   ```shell-session
+   brew tap hashicorp/tap
+   ```
+   
+1. Next, install the Consul K8s CLI with the `hashicorp/tap/consul` formulae.
```
Comment (trujillo-adam) [inline website/content/docs/k8s/installation/install.mdx:40]:
```suggestion
1. Install the Consul K8s CLI with the `hashicorp/tap/consul` formula.
```
To me, it's cleaner to remove words that signal a sequence (i.e., first, next, then, etc.), but I could go either way. Also, is there more than one formula? If so, ignore that suggestion, too.

**Review by trujillo-adam — APPROVED** [review]
Made some suggestions.

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/495742.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 6defa9ddb8e790350c941025606b4f9b41020626 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::x: Cherry pick of commit 6defa9ddb8e790350c941025606b4f9b41020626 onto `release/1.10.x` failed! [Build Log](https://circleci.com/gh/hashicorp/consul/495742)

**Pull request merged**